parts/kubernetesmastercustomscript.sh

```diff
@@ -165,6 +167,11 @@ function writeKubeConfig() {
     chmod 700 $KUBECONFIGDIR
     chmod 600 $KUBECONFIGFILE
 
+    FQDNSuffix="cloudapp.azure.com"
+    if [ "$TARGET_ENVIRONMENT" = "AzureChinaCloud" ]
```

Comment:
AFAIK, this is the only 1 fqdn need to updated. There is also a fqdn config in azureconst.go which is used to generate kube config files based on the locations. However, I found these generated kubeconfigs are not used. The kubeconfig in the destination env is generated by this sh script.